src/topology/topological_fiber_bundle.lean

```diff
@@ -733,7 +733,7 @@ instance [inhabited B] [inhabited (E (default B))] :
   inhabited (total_space E) := ⟨⟨default B, default (E (default B))⟩⟩
 
 /-- `bundle.proj E` is the canonical projection `total_space E → B` on the base space. -/
-@[simp, mfld_simps] def proj : total_space E → B :=
+@[simp, mfld_simps, reducible] def proj : total_space E → B :=
```

Comment:
I am not sure what `reducible` is supposed to do beyond typeclass inference (which is not relevant here). Can you explain what this is good for?

Reply:
@eric-wieser asked me to do it in #7803 (I asked your opinion, but you did not answer me, sigh)!

Reply:
I think `simp` and/or `rw` can see through reducible defs, but I don't remember which.